### ISSUE TITLE
load quickmarks from additional location

### ISSUE THREAD

**Issue by iambumblehead** [qutebrowser/qutebrowser#7942]
Thank you for my favorite browser, qutebrowser.

This request is to load quickmarks from an extra location, other than the default location. This would allow Guix and NixOS to apply custom lists of quickmarks to static read-only files iloaded separately.

**Comment by The-Compiler** [comment]
I don't quite follow, could you elaborate on what the goal of this is? What extra location? Why? What do you mean with "apply custom lists of quickmarks to static read-only files iloaded separately."?

**Comment by iambumblehead** [comment]
I hope this does not become too wordy but... using Guix, for example, can apply a "home" file to your system and it generates the dot files and other things needed for your user's "home environment" and when it does this, it writes read-only files then symlinks files like ~/.config/qutebrowser/config.py to those read-only version created by Guix from your.

I'd like to, for example, have something like this inside config.py
```python
config.quickmarks_include('staticurls')
```
And allow qutebrowser itself to continue adding quickmarks to ~/.config/qutebrowser/bookmarks/urls, but also reading quickmarks from 'staticurls'. Periodically one could update their Guix configuration to generate 'staticurls' with updated lists of quickmarks. I hope this description makes sense, thanks for reading me and replying.

**Comment by The-Compiler** [comment]
Thanks for the explanation, that makes more sense. Merging this into #7512.
